javers-core/src/main/java/org/javers/core/metamodel/object/CdoSnapshot.java

```diff
@@ -26,6 +26,7 @@
     private final CdoSnapshotState state;
     private final SnapshotType type;
     private final List<String> changed;
+    private final Long version;
```

Comment:
Ok, I'll change that. Do you prefer for me to simply commit this change or to squash it with the previous (I didn't find contribution guidelines)?